packages/keyring-api/src/api/balance.test.ts

```diff
@@ -0,0 +1,32 @@
+import { is } from '@metamask/superstruct';
+
+import { BalanceStruct } from './balance';
+
+describe('BalanceStruct', () => {
+  it.each([
+    // Valid
+    { balance: { amount: '1.0', unit: 'ETH' }, expected: true },
+    { balance: { amount: '0.1', unit: 'BTC' }, expected: true },
+    // FIXME: Those are not valid for `StringNumberStruct`, but they should be:
```

Comment:
Were those tests originally commented out? We should fix them if it's the case.

Reply:
I'll remove them.